src/encoded/static/components/publication.js

```diff
@@ -26,7 +28,64 @@ const PublicationComponent = (props, reactContext) => {
     ];
 
     const crumbsReleased = (context.status === 'released');
-
+    const accessions = context.datasets.reduce((dataset1, dataset2) => `${dataset1}&accession=${dataset2.accession}`, '');
+    const sort = (a, b) => a - b;
+    const datasetsColumns = {
```

Comment:
This is a pretty large assignment that I would normally put into this module’s global space so that it only gets assigned once at page load, or sometimes as a property of the class (which you don’t have since this is a functional component) if you need the object namespaced. Not a lot can cause a re-render of a Publication page, but logging in while showing a Publication page will cause this assignment to happen several times as different context and props get updated for the page. If this object used props or state directly, you’d be forced to put it here, but `SortTable` column objects are always self-contained, and any data they use gets passed at render time.

Reply:
corrected